invisible_cities/cities/components_test.py

```diff
@@ -318,3 +319,14 @@ def test_check_max_time_units():
     with raises(ValueError):
         check_max_time(max_time, buffer_length)
 
+def test_read_wrong_pmt_ids(ICDATADIR):
+    """
+    Check if an error is raised when wrong PMT IDs are read from nexus file
```

Comment:
Maybe explain that the nexus file contains one or more sensor IDs that are not present in the database, that we expect this to produce an error, and that therefore we check for that.

Reply:
Done.